src/components/AccountCreatedPage.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { useSourceApp, SourceAppLogo } from './SourceApp'
+import { Button, Link, Grid } from '@mui/material'
+import { AppContextConsumer } from 'AppContext'
+import { Typography } from 'synapse-react-client'
+import { Link as RouterLink } from 'react-router-dom'
+import SourceAppConfigs from './SourceAppConfigs'
+
+export type AccountCreatedPageProps = {}
+
+export const AccountCreatedPage = (props: AccountCreatedPageProps) => {
+  const sourceApp = useSourceApp()
+  return (
+    <>
+      <AppContextConsumer>
+        {appContext => (
+          <div className="panel-wrapper-bg AccountCreatedPage">
```

Comment:
Styling is a mix of the old way (using reusable css classes) and stylized components.